py/tests/unit/with_runtime_sparkling/test_automl.py

```diff
@@ -142,7 +143,31 @@ def testDeserializationOfUnfittedPipelineWithAutoML(classificationDataset):
     algo = setParametersForTesting(H2OAutoML()).setLeaderboardDataFrame(testingDataset)
 
     pipeline = Pipeline(stages=[algo])
-    pipeline.write().overwrite().save("file://" + os.path.abspath("build/automl_pipeline"))
-    loadedPipeline = Pipeline.load("file://" + os.path.abspath("build/automl_pipeline"))
+    pipeline.write().overwrite().save("file://" + os.path.abspath("build/automl_pipeline_leaderboardDF"))
+    loadedPipeline = Pipeline.load("file://" + os.path.abspath("build/automl_pipeline_leaderboardDF"))
     loadedPipeline.fit(trainingDateset)
 
+
+def testBlendingDataFrameHasImpactOnAutoMLStackedEnsambleModels(classificationDataset):
+    [trainingDateset, blendingDataset] = classificationDataset.randomSplit([0.9, 0.1], 42)
+
+    automl = setParametersForTesting(H2OAutoML())
+    automl.fit(trainingDateset)
+    defaultLeaderboard = truncateModelId(automl.getLeaderboard())
+
+    automl = setParametersForTesting(H2OAutoML()).setBlendingDataFrame(blendingDataset)
+    automl.fit(trainingDateset)
+    leaderboardWithBlendingDataFrameSet = truncateModelId(automl.getLeaderboard())
+
+    unit_test_utils.assert_data_frames_have_different_values(defaultLeaderboard, leaderboardWithBlendingDataFrameSet)
```

Comment:
maybe it would be better to compare only the SEs, and verify that the other models are the same (otherwise you can't prove that the difference in SEs is due to the blending frame).

Reply:
Will improve that, thank you!